okio/build.gradle

```diff
@@ -0,0 +1,29 @@
+description = 'Okio'
+
+apply plugin: 'java-library'
+apply plugin: 'osgi'
```

Comment:
Oh dear...

Reply:
Yeah, that's used for `Automatic-Module-Name`, I'll if there's a better way

Reply:
You don't actually need it to configure the jar manifest: https://docs.gradle.org/current/userguide/java_plugin.html#sub:manifest